stock_picking_invoice_link/migrations/8.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, SUPERUSER_ID
+from openupgradelib import openupgrade
```

Comment:
@pedrobaeza don't you need to declare the external dependency in __openerp__.py ?